heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -102,6 +102,7 @@ function process_message()
         msg.Fields.appBuildId       = app.buildId
         msg.Fields.vendor           = app.vendor
         msg.Fields.clientId         = parsed.clientId
+        msg.Fields.creationDate     = parsed.creationDate
 
```

Comment:
I thought about it but that would create a mismatch between the payload and the fields. Ideally we should preserve the format of the fields in the payload to avoid surprises in analyses downstream.